Directory.Build.props

```diff
@@ -46,7 +46,8 @@
           ./build/Install-WindowsSDKISO.ps1 18298
         -->
         <TargetPlatformVersion>10.0.18298.0</TargetPlatformVersion>
-        <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
+		<!-- XAML Islands require SDK 18226 -->
+        <TargetPlatformMinVersion>10.0.18226.0</TargetPlatformMinVersion>

```

Comment:
If XamlIslands requires a newer platform, then the version should be overridden in those specific `csproj` files